src/bigwig/BigWigReader.ts

```diff
@@ -51,6 +51,8 @@ interface RPLeafNode {
     dataSize: number;
 }
 
+type ParseFunction<T> = (chrom: string, startBase: number, endBase: number, rest: string, binaryParser?: BinaryParser) => T;
```

Comment:
Don't need the `binaryParser` argument here

Reply:
Done